README.md

```diff
@@ -16,6 +16,8 @@ Install prerequisites
     brew install postgresql
     bundle install
 
+Ruby gems are vendored into `vendor/cache`, you should always check in the gems when changing gems. The caching is set up with [`bundle package --all`](http://bundler.io/bundle_package.html).
```

Comment:
The docs says `--all` is remembered. I guess it is, by creating/updating `.bundle/config`. But for it to be true, for all members of the project, don't we need to commit `.bundle/config` then?